projects.json

```diff
@@ -2721,6 +2721,61 @@
       }
     ]
   },
+  {
+  "repository": "Git",
+  "url": "https://github.com/masters3d/xswift.git",
+  "path": "xswift",
```

Comment:
We like to keep projects configurations sorted alphabetically by 'path'. Please add these configurations at the end of the file, after the Vapor entry.